Sources/GRPCCore/Call/Client/ClientResponse.swift

```diff
@@ -382,4 +382,13 @@ extension StreamingClientResponse {
       return RPCAsyncSequence.throwing(error)
     }
   }
+
+  /// Returns the body parts (i.e. `messages` and `trailingMetadata`) returned from the server.
+  ///
+  /// For rejected RPCs (in other words, where ``accepted`` is `failure`), this method throws an `RPCError`.
+  public var bodyParts: RPCAsyncSequence<Contents.BodyPart, any Error> {
+    get throws {
+      try self.accepted.get().bodyParts
+    }
+  }
```

Comment:
If the caller gets this and then iterates it in two expression they'll have:

```swift
let bodyParts = try response.bodyParts
for try await part in bodyParts {
  // ...
}
```

Can we do what's done for `messages` above this so that we can avoid that? (Sorry, the issue does so "or throws"; I didn't think about it enough when filing it.)